packages/mui-base/src/useOption/useOption.types.ts

```diff
@@ -21,6 +21,10 @@ export interface UseOptionReturnValue {
    * If `true`, the option is selected.
    */
   selected: boolean;
+  /**
+   * If `true`, the component is being focused using keyboard.
+   */
+  focusVisible: boolean;
```

Comment:
Let's slowly phase out the focusVisible workaround. It's needed only for old browsers. We can add it if developers complain, but let's see if there's even a need for it.